sources/board.c

```diff
@@ -0,0 +1,69 @@
+/*
+** EPITECH PROJECT, 2022
+** Gomoku
+** File description:
+** board
+*/
+
+#include <stdlib.h>
+#include "board.h"
+
+static board_t board = {NULL, 0, 0};
```

Comment:
ew

Reply:
je vois pk mais quand même moche si c'est ton astuce pour rester dans la norme c'est quand même raté

Reply:
It's not as bad as you think. This board may be used everywhere in our project, but as a static variable, only functions which are in this file can use this variable. So I've defined some functions to use this board instead of using it directly in our project.